src/miner.cpp

```diff
@@ -528,6 +529,15 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn,  unsigned int nBlo
         return NULL;
     CBlock *pblock = &pblocktemplate->block; // pointer for convenience
 
+    // We create a temporary sidechain tx commitment builder and populate it
+    // with the txs and certs that are progressively added to the candidate block.
+    // Each time this happens we check for failures that may be caused by:
+    // - reaching the SC subtree limit (currently max 4096 sidechains)
+    // - reaching the FT / BWT / CERT / CSW limit in each subtree (currently max 4096 in each category subtree per sc)
```

Comment:
Very minor point: for `FT / BWT / CERT / CSW` the limit currently is `4095`; I wonder if it is maybe more maintainable to drop the numeric values and maybe just reference the associated constant variables names.

Reply:
Reworded in e6db47d33